core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java

```diff
@@ -299,6 +299,10 @@ public ContextMenu from(ModelObjectWithContextMenu self, StaplerRequest2 request
             return from(self, request, response, "sidepanel");
         }
 
+        public ContextMenu from(ModelObjectWithContextMenu self, StaplerRequest request, StaplerResponse response) throws JellyException, IOException {
+            return from(self, StaplerRequest.toStaplerRequest2(request), StaplerResponse.toStaplerResponse2(response), "sidepanel");
```

Comment:
Alternatively,
```suggestion
            return from(self, StaplerRequest.toStaplerRequest2(request), StaplerResponse.toStaplerResponse2(response));
```
?
?